src/common.ts

```diff
@@ -134,13 +137,19 @@ export class LoggingCommon {
     this.logName = options.logName || 'winston_log';
     this.inspectMetadata = options.inspectMetadata === true;
     this.levels = options.levels || NPM_LEVEL_NAME_TO_CODE;
-    this.stackdriverLog = new Logging(options).log(this.logName, {
-      removeCircular: true,
-      // See: https://cloud.google.com/logging/quotas, a log size of
-      // 250,000 has been chosen to keep us comfortably within the
-      // 256,000 limit.
-      maxEntrySize: options.maxEntrySize || 250000,
-    });
+    this.redirectToStdout = options.redirectToStdout ?? false;
+
+    if (!this.redirectToStdout) {
+      this.stackdriverLog = new Logging(options).log(this.logName, {
+        removeCircular: true,
+        // See: https://cloud.google.com/logging/quotas, a log size of
+        // 250,000 has been chosen to keep us comfortably within the
+        // 256,000 limit.
+        maxEntrySize: options.maxEntrySize || 250000,
```

Comment:
nit: i saw the comment about this parameter in index.ts. IMO it should be transparent to the caller. please, create an issue to handle max size in logSync using the same logic.

Reply:
Yes, this was my original concern as well - opened [1237](https://github.com/googleapis/nodejs-logging/issues/1237). Worth mentioning that when running in GCP with logging agent, it has a logic to truncate log entries, however it might not be the same way as we do in nodejs-logging